<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>bsMeasureWidget</class>
  <widget class="QWidget" name="bsMeasureWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>480</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Measure Tool</string>
    </property>
    <layout class="QGridLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item row="0" column="0">
        <widget class="QFrame" name="Frame3">
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="0" column="0">
              <widget class="QLabel" name="m_label_01">
                <property name="text">
                  <string>Start point</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="QLineEdit" name="m_start_point">
                <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                  <string>NULL</string>
                </property>
                <property name="enabled">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="QLineEdit" name="m_end_point">
                <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                  <string>NULL</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="m_label_02">
                <property name="text">
                  <string>End point</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="m_label_03">
                <property name="text">
                  <string>Distance</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="QLineEdit" name="m_distance">
                <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                  <string>NULL</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
